src/net_processing.cpp

```diff
@@ -2997,9 +2997,8 @@ void PeerManagerImpl::ProcessMessage(CNode& pfrom, const std::string& msg_type,
             return;
         }
 
-        if (vAddr.size() > MAX_ADDR_TO_SEND)
-        {
-            Misbehaving(pfrom.GetId(), 20, strprintf("%s message size = %u", msg_type, vAddr.size()));
```

Comment:
Why did the `Misbehaving` was replaced by `LogPrint` ? I took a look at `Misbehaving` and saw that it calls the `LogPrint` inside of itself. So it seems to be a logging function with some other stuff. Would explain the difference between `LogPrint` and `Misbehaving`?

Reply:
This is explained in the pull request description. Practically, I couldn't find a difference between `Misbehaving` and `LogPrint` on one of my nodes. In theory there is a difference, but I explain why it shouldn't matter. (The difference is that `Misbehaving` may disconnect and discourage the peer, unless it is a manual or noban connection).

Reply:
Thank you, @MarcoFalke I'm new to bitcoin source code and have lots of questions, posting my basic questions as comment on PRs good way to ask my questions? Or there are better places? such as IRC or bitcoin-talk? I'm aware that this question has nothing to do with the PR but I couldn't find a way to send direct message to you.

Reply:
You can also ask on the dev IRC (https://github.com/bitcoin/bitcoin/blob/master/CONTRIBUTING.md#communication-channels), the review IRC (https://bitcoincore.reviews/), or on Bitcoin StackExchange.